### ISSUE TITLE
enhancement: Discussion on post-endgame content

### ISSUE THREAD

**Issue by mefistotelis** [swfans/swars#73]
Currently, after the last mission ends, and outro is played, the player ends up with a desktop without further emails. In Eurocorp campaign it is possible to leave two emails unfinished, so they may be still there if the player knew to save them; otherwise, there's nothing.

But it is possible to do more there. My ideas:

1. Give the player an email with congratulations and ability to start a mission in any city from the campaign (if there were two missions in a city, only one (the later one?) would be available). The mission would be its own success trigger - so after finishing, the player would get it again.

2. Give the player an email with congratulations and possibility to play cut content for that campaign - selected few best missions from the cut content. The cut missions would be later improved to be more fun, when we have an editor. They can be available all at one, or in form of mini-campaign, with each having a short briefing.

**Comment by Moburma** [comment]
Yeah, this is interesting, I don't personally have a strong preference for what to do yet. I guess the obvious one would be to give the player the demo levels at least - the Zealot one is quite interesting and unique (especially as it was not available before), although the Eurocorp one is considerably less so (it's just a slightly different version of mission 36 after all). They do already have briefings, after all.

How about maybe a "New Game+" mode? Basically starting the campaigns again only with the current research and equipment. That said I'm not sure how fun that would be as the early levels are a whitewash with end game weapons.

As an aside I wonder if one reason the game works as it does is because my personal hunch is you could switch between campaigns with that weird S shaped thing visible in early screenshots. There's no campaign selector in the pre-alpha menus, so I think this was how it was done. No real evidence for this, mind.

**Comment by mefistotelis** [comment]
I remember when I finished the game and seen the empty desktop, I was kind of disappointed. Ability to play again from the beginning probably wouldn't improve that. I don't think playing the same but overpowered would be interesting to anyone.

But ability to select and play any level - this does feel like an award. Maybe there is that one level which caused the player grief and he would be happy to go there again, this time beefed to max and with Grav Gun.

**Comment by geist22** [comment]
I would just leave the basic campaigns as they are, and leave such changes for other places, particularly the extended campaign you hinted at previously.

> 1. Give the player an email with congratulations and ability to start a mission in any city from the campaign (if there were two missions in a city, only one (the later one?) would be available). The mission would be its own success trigger - so after finishing, the player would get it again.

This sounds similar to a mechanic from the first Syndicate game, where the population of a certain country would rebel if you taxed them too much, and you'd then lose that country to an enemy syndicate and have to redo the mission to bring it back under your control. 
But it doesn't make much sense for SWARS, also I believe that the power creep in SWARS is too steep and would make most missions trivial with more advanced tech.

Replaying missions could be enabled in your future extended campaigns, where enemies could have better weapons/mods to keep up with your research level, to keep the missions from becoming a trivial stroll.

> 2. Give the player an email with congratulations and possibility to play cut content for that campaign - selected few best missions from the cut content. The cut missions would be later improved to be more fun, when we have an editor. They can be available all at one, or in form of mini-campaign, with each having a short briefing.

I'd like the cut content to be exposed in the menu in some way (as opposed to requiring -m start from command line), but adding those missions at the end would just result in the player rushing through with the Plasma Lance. As you know I'm guilty of saving EuroCorp missions 3 and 10 to research the Graviton Gun after the last mission, so I know just how much of a massacre these missions can be with highly upgraded agents (even without Plasma Lances).

If you want to go that road, you could inject those missions as optional (like EuroCorp 3 and 10) ones in your planned extended campaigns in places where it would make sense difficulty- and equipment-wise. 

Personally, I think a good place to put the cut missions would be like some games do it with expansion packs: at the login screen you have a new menu option, where you would find a list of any mission not found in the campaign, and could just play individual missions from there without commiting to a full campaign. That would require forcing a standard loadout rather than letting the player decide what to bring.
